Update/onik_002.txt

```diff
@@ -2160,7 +2157,7 @@ void main()
 //　レナはこれ以上ないくらいに赤面しながら、信じられない…という表情を浮かべる￥
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　レナはこれ以上ないくらいに赤面しながら、信じられない…という表情を浮かべる。",
-		   NULL, "Rena turned an even deeper shade of red, the words \"I can't believe it\" practically written on her face.", Line_Normal);
+		   NULL, "Rena turned an even deeper shade of red, the words \"I can't believe it!\" practically written on her face.", Line_Normal);
```

Comment:
I don't think this is correct.  The exclamation mark forces more energy into the phrase than may be intended.